src/main/java/Otto.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Otto {
+    private final TaskList taskList;
+
+    public Otto() {
+        this.taskList = new TaskList();
+    }
+
+    private void intro() {
+        System.out.println((OttoResponses.owl + OttoResponses.intro).indent(4));
+        printLine();
+    }
+
+    private void printLine() {
+        System.out.println(OttoResponses.line.indent(4));
+    }
+
+    private void exit() {
+        this.displayMsg(OttoResponses.bye);
+    }
+
+    private void displayMsg(String msg) {
+        printLine();
+        System.out.println(msg.indent(4));
+        printLine();
+    }
+
+    private void displayTaskList() {
```

Comment:
Perhaps tasks related methods could be placed in the TaskList.java file rather than in this file